src/components/pod/pod.test.js

```diff
@@ -3,7 +3,53 @@ import Pod from "./pod.component";
 import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
 import Content from "../content";
 
-import { podBlock } from "../../../cypress/locators/pod";
+import {
+  podBlock,
+  podComponent,
+  divPodComponent,
+  podTitle,
+  podSubTitle,
+  podFooter,
+  podSoftDelete,
+  podContent,
+  podEditContainer,
+  podDelete,
+  podUndo,
+  podEdit,
+} from "../../../cypress/locators/pod";
+
+import { useJQueryCssValueAndAssert } from "../../../cypress/support/component-helper/common-steps";
+
+const specialCharacters = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+
+const PodComponent = ({ ...props }) => {
+  return (
+    <Pod
+      title="Title"
+      subtitle="Subtitle"
+      footer="Footer"
+      onEdit={() => {}}
+      onDelete={() => {}}
+      mb={3}
+      {...props}
+    />
+  );
+};
+
+const SoftDeletePodComponent = ({ ...props }) => {
```

Comment:
Personally I would either add the `softDelete` prop to the internal pod by default and amend the affected tests, or rename this mock component, since this is a bit misleading currently as this is not a render soft delete pod by default.